requirements.development.txt

```diff
@@ -0,0 +1,2 @@
+py==1.4.32
```

Comment:
Not sure there but it is some dependency of `pytest` I guess. This file was the result of `pip freeze` just after `pip install pytest==3.0.6` in a clean environment.